src/logion/controllers/locrequest.controller.ts

```diff
@@ -548,24 +551,35 @@ export class LocRequestController extends ApiController {
 
     static closeLoc(spec: OpenAPIV3.Document) {
         const operationObject = spec.paths["/api/loc-request/{requestId}/close"].post!;
-        operationObject.summary = "Closes a LOC";
+        operationObject.summary = "Closes a LOC, and optionally provides a seal.";
         operationObject.description = "The authenticated user must be the owner of the LOC.";
-        operationObject.responses = getDefaultResponsesNoContent();
+        operationObject.responses = getDefaultResponses("SealView");
```

Comment:
I think it is `CloseView` instead of `SealView`.

Reply:
Indeed, thanks for the close look !